.github/workflows/build.yml

```diff
@@ -143,7 +142,7 @@ jobs:
         include:
           # macOS
           - os: macos-13
```

Comment:
Could we bump this to `macos-latest` too? iirc, the reason we were using `macos-13` was because Qt5 didn't work on ARM macs.

Reply:
I think the reason for macos-13 was some C++20 code not working in https://github.com/Chatterino/chatterino2/pull/4970 (See third bullet point of description)
I don't remember which one, tho.

Reply:
That was because [`macos-latest` resolved to `macos-12`](https://web.archive.org/web/20231123025253/https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners) at the time. So we should be fine with using latest.

Reply:
Will do this in a follow-up PR